app/build.gradle.kts

```diff
@@ -55,13 +55,23 @@ android {
     buildTypes {
         debug {
             manifestPlaceholders["targetBitwardenAppId"] = "com.x8bit.bitwarden.dev"
+            buildConfigField(
+                type = "com.bitwarden.authenticatorbridge.manager.model.AuthenticatorBridgeConnectionType",
+                name = "AUTHETNICATOR_BRIDGE_CONNECTION_TYPE",
```

Comment:
Small typo here
```suggestion
                name = "AUTHENTICATOR_BRIDGE_CONNECTION_TYPE",
```

Reply:
Thanks :) I'll blame the typo ALL_CAPS 😆 😉